modules/manage/pages/kubernetes/k-manage-resources.adoc

```diff
@@ -27,12 +27,32 @@ kubectl describe nodes
 [[memory]]
 == Configure memory resources
 
-On a worker node, Kubernetes and Redpanda processes are running at the same time, including the Seastar subsystem that is built into the Redpanda binary. Each of these processes consumes memory. You can configure the memory resources that are allocated to these processes.
+On a worker node, Kubernetes and Redpanda processes are running at the same time, including the Seastar subsystem that is built into the Redpanda binary. Each of these processes consumes memory. Proper configuration of memory resources ensures optimal performance and stability for your Redpanda deployment.
 
-By default, the Helm chart allocates 80% of the configured memory in `resources.memory.container` to Redpanda, with the remaining reserved for overhead such as the Seastar subsystem and other container processes.
-Redpanda Data recommends this default setting.
+=== Memory allocation and Seastar flags
```

Comment:
OOC do we talk about resource allocation on baremetal? Last I did a search the only instances of `--memory` and the like showed up in our Kubernetes docs which should ideally be abstracting those details away 😅

Reply:
We have a Jira to document the relevant Seastar flags for bare-metal deployments. But since we don't have that yet, I wanted to minimally explain the two flags that affect the two memory management options in K8s.

Reply:
Ah gotcha. In that case I'm going to defer to either @dotnwat or @travisdowns on the descriptions here to avoid saying anything incorrect. My mental model of `--reserve-memory` is that it's needlessly convoluted which we probably don't want to put in our docs.

Reply:
@JakeSCahill @chrisseto can I get some background/context on this doc? Why do we document the `--memory` and `--reserve-memory` flags here? If anything they could be documented in a non-k8s part of the doc, or maybe we don't have that.

I'm happy to give precise definitions of these flags (I don't think the below is right), but let's first decide if we want to document them and in what context.

Reply:
> We have a Jira to document the relevant Seastar flags for bare-metal deployments. But since we don't have that yet, I wanted to minimally explain the two flags that affect the two memory management options in K8s.

@travisdowns, we're documenting the flags here because we don't yet have docs for them. Since we have two ways of setting memory resources in Kubernetes, and the differences are in how we configure those flags, I thought it would be useful to explain them to users.

We could gloss over the differences and say that one is legacy and not for production, but for completeness, it might be better to explain the reasons.

Reply:
@JakeSCahill sure makes sense. These flags are bit hard to give independent definitions for because the behavior of one depends on whether the other is set, so there's really a 2x2 matrix of {not set, set} for each flag we need to consider.

This is the write-up of the behavior of these two flags I refer to and try to keep up to date:

https://github.com/scylladb/seastar/issues/375#issuecomment-2530012089

I just now added the 2x2 matrix explicitly. Does that give enough detail? Does it work for your documentation purposes?

Reply:
That's great. I've added some of this extra info to a collapsible block for those who want the details.